src/components/Create/components/pages/ProjectToken/hooks/ProjectTokenForm.ts

```diff
@@ -202,5 +210,15 @@ export const useProjectTokensForm = () => {
     },
   })
 
+  useFormDispatchWatch({
```

Comment:
Consider using `ignoreUndefined: true` just in case - was there before 🤷‍♂️

Reply:
Done!